### PR TITLE
[PLT-6996] On "About Mattermost", add notes with links to notice.txt which has list of copyrights of open source components used

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
On "About Mattermost", add notes with links to notice.txt which has list of copyrights of open source components used

#### Ticket Link
Jira ticket: [PLT-6996](https://mattermost.atlassian.net/browse/PLT-6996)

#### Checklist
- [x] Has UI changes
screen shot 2017-08-29 at 10 12 37 pm

@asaadmahmood above screenshot is the closest I could get, please adjust UI if necessary. thanks!

### CODE REVIEW

File: webapp/components/about_build_modal.jsx [hunk 1]
```diff
@@ -167,7 +167,15 @@ export default class AboutBuildModal extends React.Component {
                             />
                         </div>
                     </div>
-                    <div className='about-modal__hash form-group padding-top x2'>
+                    <div className='about-modal__notice form-group padding-top x2'>
+                        <p>
+                            <FormattedHTMLMessage
+                                id='about.notice'
+                                defaultMessage='Mattermost is made possible by the open source software used in our <a href="https://about.mattermost.com/platform-notice-txt/" target="_blank">platform</a>, <a href="https://about.mattermost.com/desktop-notice-txt/" target="_blank">desktop</a> and <a href="https://about.mattermost.com/mobile-notice-txt/" target="_blank">mobile apps</a>.'
```
Comment (hmhealey) [inline webapp/components/about_build_modal.jsx]: Small thing. I don't think "apps" is supposed to be part of the "mobile" link

Comment (saturninoabril) [reply webapp/components/about_build_modal.jsx]: @hmhealey actually, the rough design and the description did not match in Jira ticket. I could easily change it but honestly, I think apps should be included on link.

Comment (saturninoabril) [reply webapp/components/about_build_modal.jsx]: @hmhealey updated the link

**Comment by saturninoabril** [comment]
Thanks @asaadmahmood !

**Pull request merged**